frontend/packages/policy-editor/src/types/index.ts

```diff
@@ -18,6 +19,51 @@ export interface PolicySubject {
   subjectDescription: string;
 }
 
+export interface PolicyAccessPackage {
+  id: string;
+  urn: string;
+  name: string;
+  description: string;
+}
+
+export interface PolicyAccessPackageArea {
+  id: string;
+  urn: string;
+  name: string;
+  description: string;
+  icon: string;
+  areaGroup: string;
+  packages: PolicyAccessPackage[];
+}
+
+export interface PolicyAccessPackageAreaGroup {
+  id: string;
+  urn: string;
+  name: string;
+  description: string;
+  type: string;
+  areas: PolicyAccessPackageArea[];
+}
+
+export interface AccessPackageResource {
+  identifier: string;
+  title: {
+    nb: string;
+    nn: string;
+    en: string;
```

Comment:
These three lines are duplicated below. I suggest making a type for them (if it is not already existing) 😄